bundles/aliyun/src/main/java/org/apache/gravitino/oss/fs/OSSFileSystemProvider.java

```diff
@@ -61,9 +66,22 @@ public FileSystem getFileSystem(Path path, Map<String, String> config) throws IO
     }
 
     hadoopConfMap.forEach(configuration::set);
+
     return AliyunOSSFileSystem.newInstance(path.toUri(), configuration);
   }
 
+  @Override
+  public Map<String, String> getFileSystemCredentialConf(Credential[] credentials) {
+    Credential credential = OSSCredentialsProvider.getSuitableCredential(credentials);
```

Comment:
Could you move `getSuitableCredential` to an util class?  It's odd to invoke `OSSCredentialsProvider` here